slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -51,6 +52,14 @@ var (
 			Buckets: prometheus.LinearBuckets(0.5, 0.25, 50),
 		},
 	)
+
+	PodFullStartupLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    PodFullStartupLatencyKey,
+			Help:    "Pod e2e startup latencies in milliseconds, without image pull times",
```

Comment:
Please fix this description. RIght now it's saying the same as the other metric.

Reply:
Done.